changelog.md

```diff
@@ -1,5 +1,62 @@
 
 # OpenId Connect Generic Changelog
+**3.7.3**
```

Comment:
Could you update the changelog. Also, update the plugin version, so it is easy to tell which version is installed.